### PR TITLE
Add com.markopejic.downloader

### PR DESCRIPTION
**Pull request by markop404**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
  A graphical app for downloading video/audio from hundreds of websites.

- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: com.markopejic.downloader.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  <url type="homepage">https://downloader.markopejic.com</url>
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+
+  <developer id="com.markopejic">
+    <name>Marko Pejić</name>
+  </developer>
+  <url type="vcs-browser">https://github.com/markop404/media-downloader</url>
+
+  <releases type="external">com.markopejic.downloader.releases.xml</releases>
```
Comment (hfiguiere) [inline com.markopejic.downloader.metainfo.xml:19]: this is not supported. Put the releases inline.

File: com.markopejic.downloader.json [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        "/man",
+        "/share/doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "cpython",
```
Comment (hfiguiere) [inline com.markopejic.downloader.json:27]: Python is in the runtime.

File: com.markopejic.downloader.desktop
Comment (hfiguiere) [inline com.markopejic.downloader.desktop:1]: this file should be upstream

File: com.markopejic.downloader.metainfo.xml
Comment (hfiguiere) [inline com.markopejic.downloader.metainfo.xml:1]: this file should be upstream

File: com.markopejic.downloader.desktop
Comment (hfiguiere) [reply com.markopejic.downloader.desktop:1]: it's never referenced anyway.

File: com.markopejic.downloader.metainfo.xml
Comment (hfiguiere) [reply com.markopejic.downloader.metainfo.xml:1]: it's never referenced anyway.

File: com.markopejic.downloader.json [hunk 3]
```diff
@@ -0,0 +1,120 @@
+{
+    "id": "com.markopejic.downloader",
```
Comment (hfiguiere) [inline com.markopejic.downloader.json:2]: `markopejic.com` lead to a 403 forbidden.

File: com.markopejic.downloader.metainfo.xml (hunk 1 above)
Comment (markop404) [reply com.markopejic.downloader.metainfo.xml]: [Embedded](https://github.com/markop404/flathub/commit/4d0891db6692ef15d91c0ce440107f45dde0e30d) release information directly to com.markopejic.downloader.metainfo.xml.

File: com.markopejic.downloader.json (hunk 2 above)
Comment (markop404) [reply com.markopejic.downloader.json]: Removed the module in [69f4640](https://github.com/markop404/flathub/commit/69f46402002da193d6f75e74de82f5ad06cfb043)

File: com.markopejic.downloader.json (hunk 3 above)
Comment (markop404) [reply com.markopejic.downloader.json:2]: Fixed the 403 forbidden error, the website is now reachable.

File: com.markopejic.downloader.desktop
Comment (markop404) [reply com.markopejic.downloader.desktop:1]: Fixed, this file is now upstream.

File: com.markopejic.downloader.metainfo.xml
Comment (markop404) [reply com.markopejic.downloader.metainfo.xml:1]: Fixed, this file is now upstream.

**Comment by hfiguiere** [comment]
bot, build com.markopejic.downloader

**Pull request closed without merging**